core/src/main/java/com/graphhopper/GraphHopper.java

```diff
@@ -882,11 +920,16 @@ public void postProcessing() {
         }
 
         initLocationIndex();
+
         if (chFactoryDecorator.isEnabled())
             createCHPreparations();
+        if (!isCHPrepared())
+            prepareCH();
+
+        if (lmFactoryDecorator.isEnabled())
+            lmFactoryDecorator.createPreparations(ghStorage, traversalMode);
 
-        if (!isPrepared())
-            prepare();
+        prepareLM();
```

Comment:
Yes I saw that later on :). Not sure if we should change this though. Maybe rename the perpare method to prepareIfNotPrepared for ch an lm. This would make the code a bit more readable IMHO. WDYT?